### ISSUE TITLE
ConversionException: Cannot convert node http://purl.obolibrary.org/obo/OBI_0000220 to OntClass: it does not have rdf:type owl:Class or equivalent

### ISSUE THREAD

**Issue by arteymix** [PavlidisLab/Gemma#804]
To reproduce:

```http
GET /rest/v2/datasets/categories?filter=&excludedCategories=H4sIAAAAAAAAA43RywrCMBAF0B%2BSZDJ56lKpj1U%2FoTRSbbGQElqCf29w4dKZ2Q0cLpeZcV2Xg5TLlmeRYpqnmPv8Fik%2FZV3l6docbx1q7fxu%2FE8vbQcA3ingyaDsT5ZSxBAn0d%2FF9pLDI8nm%2FEWgLdLIKEaSwUAjC56BUDOQYRR3VtHIoyGRMuios7f1k6ABw56ub%2BqQyIJzHOQ%2F0oZS9GUCAAA%3D&excludedTerms=H4sIAAAAAAAAA5XTTW%2BDMAwG4D80Jdj5INltnbqph62Txr0KiLXVoKkYiO3fDw11cEE2uSE9Mo7z%2BtS213spr11TiZjH6pw3ofkRsTnK4VO%2B7w%2FJcNCld6dRHsu6DqL%2BqkSXF6IIMl5amT1vX0YJTKeYTjPdf39LN9lvdn8wQcOliAlF3x6y8f%2BJdiusZ1u0wLZekVO4WYCU3S8oz66LQ2W2hel1SauM4tdFvNm%2B70WZn0UoRPcpy48ot0%2FjsIylEehp%2BsvIaksirafYLSNv6J60T1lIkciAJTPw%2BrjZZeE7Xg4w5ItchzkHsCs4JA7XcEXv5pxbJOMz4%2BjWFLeKXOdJI6bOrPCzmDKwo7dv0sNM%2FC9I68ot%2FwUAAA%3D%3D HTTP/1.1
```

**Comment by arteymix** [comment]
There should be a check in baseCode and a warning to avoid this every time we attempt to perform a conversion.

**Comment by arteymix** [comment]
I'm not sure this is worth fixing because it occurs rarely and is almost certainly a bug in the source ontology that needs to be fixed. I'll search for the term in the files we have.

**Comment by arteymix** [comment]
Fixed in dd763fed86b40d4e283c65f40fd2c5001c2b73c8. I've addressed the root cause in baseCode.
